en/peripherals/serial_configuration.md

```diff
@@ -1,14 +1,15 @@
 # Serial Port Configuration
 
-Most serial (UART) ports on a Pixhawk board can be fully configured via parameters
-(exceptions are ports that are used for a very specific purpose like RC input, or which are not configurable like `SERIAL 5`).
+Many serial (UART) ports on a Pixhawk board can be fully configured via parameters: e.g.: `GPS1`, `TELEM1`, `TELEM2`, `TELEM4` (`UART+I2C`).
 
 The configuration makes it easy to (for example):
 - change the baudrate on a port.
 - run MAVLink on a different port, or change the streamed messages.
 - setup dual GPS.
 - enable sensors that run on a serial port, such as some [distance sensors](../sensor/rangefinders.md).
 
+> **Note** Some ports cannot be configured (e.g. `SERIAL 5`), for example because they are used for a very specific purpose like RC input.
```

Comment:
Why is `SERIAL5` not configurable? Off top of your head, any others that are never configurable (we might as well list them here).

Reply:
Serial 5 is usually the NuttX debug console.